Example/ViewController.swift

```diff
@@ -88,6 +88,9 @@ class ViewController: UITableViewController {
         if let cell = tableView.dequeueReusableCell(withIdentifier: "MiniAppCell", for: indexPath) as? MiniAppCell {
             let miniAppDetail = self.decodeResponse?[indexPath.row]
             cell.titleLabel?.text = miniAppDetail?.displayName
+            if (miniAppDetail?.displayName) == nil {
+                cell.titleLabel?.text = "Null"
```

Comment:
what do you think of
`cell.titleLabel?.text = miniAppDetail?.displayName ?? "Null"`

Reply:
Done